Doc/library/stdtypes.rst

```diff
@@ -5207,7 +5207,8 @@ instantiated from the type::
 Other Built-in Types
 ====================
 
-The interpreter supports several other kinds of objects. Most of these support
+The interpreter supports several other kinds of objects, as well as direct
+instances of :class:`object`, the universal type. Most of these support
```

Comment:
Is there a more conventional/Pythonic term for this than "universal type"? Maybe "root base class", or something like that?

Reply:
The technical term is "top type" (https://en.wikipedia.org/wiki/Top_type). Wikipedia lists "universal type" as a synonym.

Reply:
I'm just reviewing open PRs and wondering if this one is still a thing?

Reply:
> I think the new term 'direct instances' only adds confusion. Same for 'the <???> type'. Describing 'object' itself is part of its built-in function entry. What should be said, after the existing 'few operations' sentence, is something about the 'special operations/methods' it provides.

@terryjreedy I couldn't find the phrase 'several operations'. Can you explain in more detail what you mean?

Reply:
I agree _universal type_ didn’t really work here. FWIW I’ve never come across the term _top type_, other than the Wikipedia article, and the term’s meaning is not self-explanatory to me. (_Bottom type_ would parallel _base class_ better to me.)

Maybe all we need to say here is something like “The interpreter supports several other kinds of objects, including instances of the base class _object_ itself”? But if other people are happy with _top type_, I don’t mind.

@furkanonder Maybe Terry meant the sentence following the location of CAM’s comment: “Most of these [other types] support only one or two operations.”